src/main/java/com/northconcepts/datapipeline/examples/cookbook/GenerateRecordSerializersExample.java

```diff
@@ -0,0 +1,15 @@
+package com.northconcepts.datapipeline.examples.cookbook;
+
+import com.northconcepts.datapipeline.foundations.tools.GenerateRecordSerializers;
+
+import java.util.HashSet;
+
+public class GenerateRecordSerializersExample {
+    public static void main(String[] args) throws Throwable {
+        GenerateRecordSerializers serializers = new GenerateRecordSerializers();
+        HashSet<String> types = new HashSet<>();
+        types.add("com.northconcepts.datapipeline.foundations.datamapping.DataMapping");
+        types.add("com.northconcepts.datapipeline.foundations.pipeline.input.CsvPipelineInput");
```

Comment:
Let's create a couple static inner classes in this class to replace these two.  Let's make them simple POJOs -- nothing too complicated that hides the main purpose of the example.

Reply:
done